### ISSUE TITLE
Plugin error output not shown in workflow logs

### ISSUE THREAD

**Issue by dustinblack** [arcalot/arcaflow-engine#31]
## Describe the bug

If a plugin encounters an internal error during a workflow, the engine output implies it is displaying the error output from the plugin, but nothing is displayed.

```
...
2022-11-18T09:42:46+01:00	info		Step uperf_client is now executing ATP...
2022-11-18T09:42:46+01:00	debug		Executing step uperf...
2022-11-18T09:42:46+01:00	debug		Step uperf started, waiting for response...
2022-11-18T09:42:49+01:00	debug		Step uperf completed with output ID 'error'.
2022-11-18T09:42:49+01:00	info		Step uperf_client has finished with output error.
2022-11-18T09:42:49+01:00	warning		Step uperf_client had error output:
        <<---- It looks like the error output is expected here, but there is nothing ---->>
2022-11-18T09:42:49+01:00	debug		Removing dependency tree node steps.uperf_client...
2022-11-18T09:42:49+01:00	debug		Removing dependency tree node steps.uperf_client.outputs.error...
2022-11-18T09:43:44+01:00	debug		Step uperf_server completed with output ID 'success'.
2022-11-18T09:43:44+01:00	info		Step uperf_server has finished with output success.
2022-11-18T09:43:44+01:00	debug		Removing dependency tree node steps.uperf_server...
2022-11-18T09:43:44+01:00	debug		Removing dependency tree node steps.uperf_server.outputs.success...
2022-11-18T09:43:44+01:00	error		Workflow execution failed (workflow run failed, output has unsuccessful dependencies: steps.uperf_client.outputs.success)
exit status 1
```

## To reproduce

As of this writing, if you use the workflow from [workflows PR #6](https://github.com/arcalot/arcaflow-workflows/pull/6) with `uperf_kbytes: 64` and `uperf_protocol: udp` in the `input.yaml` file, the uperf plugin will fail with the above reported output.

**Comment by jaredoconnell** [comment]
I think a good way to solve this problem without hard-coding stuff would be to add a config option to specify output names to print to the logs, with the log type.
For example, log complete output of "error" output as "error"

**Comment by dustinblack** [comment]
Fixed with #48
